modules/configuration/templates/form_subproject.tpl

```diff
@@ -23,31 +23,39 @@
             <fieldset>
                 <input type="hidden" name="subprojectID" value="{$subprojectID}" class="subprojectID">
                 <div class="form-group">
-                    <label class="col-sm-12 col-md-4">Subproject Title</label>
-                    <div class="col-sm-12 col-md-8">
+                    <div class="col-sm-12 col-md-3" data-toggle="tooltip" data-placement="right" title="{'Full descriptive title of the subproject'}">
+                        <label class="col-sm-12 control-label">Subproject Name</label>
+                    </div>
+                    <div class="col-sm-12 col-md-7">
```

Comment:
change these 7s to 9s

Reply:
and this (https://github.com/colezweber/Loris/blob/0117bd2be8b5e4e5b86e66124245bdd43a81b427/modules/configuration/templates/form_subproject.tpl#L16) to 7